package/MDAnalysis/analysis/gnm.py

```diff
@@ -37,21 +37,22 @@
 
 An example is provided in the MDAnalysis Cookbook_, listed as GNMExample_.
 
-.. GNMExample_: https://github.com/MDAnalysis/MDAnalysisCookbook/blob/master/examples/GNMExample.py
-.. Cookbook_: https://github.com/MDAnalysis/MDAnalysisCookbook
+.. _GNMExample: https://github.com/MDAnalysis/MDAnalysisCookbook/blob/master/examples/GNMExample.py
+.. _Cookbook: https://github.com/MDAnalysis/MDAnalysisCookbook
 
 The basic approach is to pass a trajectory to :class:`GNMAnalysis` and then run
 the analysis::
 
-   u = MDAnalysis.Universe(PSF, DCD)
-   C = MDAnalysis.analysis.gnm.GNMAnalysis(u, ReportVector="output.txt")
+    u = MDAnalysis.Universe(PSF, DCD)
+    C = MDAnalysis.analysis.gnm.GNMAnalysis(u, ReportVector="output.txt")
+
+    C.run()
+    output = zip(*C.results)
 
-   C.run()
-   output = zip(*C.results)
+    with open("eigenvalues.dat", "w") as outputfile:
+        for item in output[1]:
+            outputfile.write(item + "\n")
```

Comment:
No this is normal python behavior. `\n` is an escape sequence and just like for docs with a latex string inside we need to tell python the string is a raw string.